docs/versionIndex.md

```diff
@@ -385,7 +385,7 @@ backend services have been tested with each released version of ADF.
 -   [Info drawer tab component](core/components/info-drawer-tab.component.md)
 -   [Node permission dialog service](content-services/services/node-permission-dialog.service.md)
 -   [Search check list component](content-services/components/search-check-list.component.md)
--   [Search date range component](content-services/components/search-date-range.component.md)
+-   Search date range component
```

Comment:
I'm wondering what we should to do with that. This is single component without documentation and even if that component is listed user won't be able to open documentation for it. @swapnil-verma-gl @MichalKinas what do you think?

Reply:
I agree that this seem like an odd case. The reason why I didn't remove it is because this is essentially a history of what was released in which version. Removing it here would be acting like the component was never added into the repository, which does not seem right. I removed the link as it would now point to the new component instead of the old one (the documentation for it is removed)
We can reintroduce the link, have it redirect to the new component, and then maybe add a note in the documentation there that the new component is meant to be the replacement of the old date-range component. We can also add another note in the most latest version in versionIndex, stating that the old date-range component was replaced with the new one

Reply:
I see also 3rd option - we can put there link to that old file in github (like to commit which removes that file so user can find old documentation in that commit).

To be honest I'm not sure how to approach that so let's leave it for now unless @MichalKinas knows how we approach this kind of situations.

Reply:
Okay so I think that the best solution in this case would be adding in this line the info that this component has been replaced by the new one (with link to the docs for the new one) and the info in which version the change happened

Reply:
@swapnil-verma-gl please do as suggested by @MichalKinas

Reply:
The problem is the name of the 'new' component is also SearchDateRange component now. This is because we 
1. Removed the old date range component
2. Renamed SearchDateRangeAdvanced component to SearchDateRangeComponent

So if I were to do the proposed change, we would have 
- Search Date Range Component - replaced by Search Date Range Component in v6.4.0

Also, just FYI, under [v6.2.0](https://github.com/Alfresco/alfresco-ng2-components/pull/9023/files#diff-6825824005472290de3d91fdefb2061938ad16c97c550d35b5fba0ea47a7eb00R60), there was the entry for SearchDateRangeAdvanced component. This has now been renamed to SearchDateRange component, and links to the new documentation. Would it be a good idea to add 2 links linking to the same documentation?

Reply:
Ok @swapnil-verma-gl so maybe we can add the same link to both places and just add some comments which explains why there are duplicated entries.

Reply:
Reintroduced this link, and added a NOTE section to the SearchDateRange component documentation mentioning that there have been changes after v6.5.0

Reply:
ok thank you